guides/common/attributes-satellite.adoc

```diff
@@ -1,4 +1,5 @@
 // Attributes only for satellite build
+:install-os: RHEL
```

Comment:
Alternative: "{RHEL}". Your call.

Reply:
Technically now it's unsed and for titles the whole name is very long. So I chose the short one.

Reply:
Works for me!